build_artifacts/v1/v1.2/v1.2.0/cpu.env.in

```diff
@@ -35,3 +35,4 @@ conda-forge::sagemaker-jupyterlab-extension[version='>=0.1.7,<1.0.0']
 conda-forge::sagemaker-jupyterlab-emr-extension[version='>=0.1.7,<1.0.0']
 conda-forge::amazon-sagemaker-jupyter-scheduler[version='>=3.0.2,<4.0.0']
 conda-forge::jupyter-server-proxy[version='>=4.1.0,<5.0.0']
+conda-forge::amazon-codewhisperer-jupyterlab-ext [version='>=2.0.0,<3.0.0']
```

Comment:
Concerned about the added space character between the name and version bracket. Does this parse successfully for the build?

Reply:
Removing this space.